packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -297,6 +309,7 @@ class RecoveryPhoneHandler {
               uid,
             }
           );
+          return { phoneNumber, nationalFormat };
```

Comment:
I think this is the problem... Missing `{ status: RecoveryPhoneStatus.SUCCESS }`